src/functions/Functions.php

```diff
@@ -84,13 +100,20 @@ public function countArguments(): array
      * !!! you need to test only exceptional case, since the behavior of countArguments was already tested in the task above
      *
      * You will need to use "Argument unpacking via ..." to pass the parameters to wrapped function:
+     *
      * @see https://www.php.net/manual/en/migration56.new-features.php#migration56.new-features.splat
      *
      * @return array
      * @throws \InvalidArgumentException
      */
     public function countArgumentsWrapper(): array
     {
-        // put your code here
+        foreach (func_get_args() as $val) {
+            if (!is_string($val)) {
+                throw new \InvalidArgumentException('countArgumentsWrapper() function accepts strings only');
+            }
+        }
+        return $this->countArguments(...func_get_args());
+
     }
 }
```

Comment:
please add empty line at the end of file

Reply:
Done